### DISCUSSION TITLE
Album Art Only Appears to show on some clients

### DISCUSSION THREAD

**Discussion by noah1234j** [kagemomiji/airsonic-advanced#372] [Q&A] [answered]
I'm sure there is something simple I am doing wrong but haven't been able to track it down. The album art is not displaying on the album but it does display for individual tracks. The art is currently in the folder as a .jpg folder along with the mp3 tracks.

I didn't have this issue when using the main branch of airsonic but now that I am on your branch it doesn't seem to be working for me.

I would also like to add thank you for keeping this project alive. You're doing great work here.
![Screenshot_20240215_215221](https://github.com/kagemomiji/airsonic-advanced/assets/24739971/f67d4023-b849-46d6-b2d2-b70dcdb1cb57)
![Screenshot_20240215_215158_substreamer](https://github.com/kagemomiji/airsonic-advanced/assets/24739971/e498f0bc-264e-49f8-876f-58d860b4d2ed)
![Screenshot_20240215_215149_Ultrasonic](https://github.com/kagemomiji/airsonic-advanced/assets/24739971/db4f5617-4762-4eff-bab6-5172ee5fd160)

**Comment by jr1234jr** [comment]
Hi,

it is exactly the same issue for me.
I also have stored artworks as folder.jpg in same folders where music files are stored.
I can see artworks in the original Web UI and I can also see artworks in all clients in the Now Playing Screen.

However, artworks are NOT when browsing album lists. Tested for following clients:

play:sub (iOS)
Ultrasonic (Android)
sonixd (Linux)

What could be the issue here? Does NowPlaying screen fetch other images than Album lists do?

Also from my side many thanks for keeping Airsonic alive :)

**Reply by Xet2016** [reply]
Hi,
It's not working on

Substreamer (iOS) and (Android)
Airsonic-Refix (PC)

I also tried to move back to an earlier working image, but now it's not working.

**Comment by noah1234j** [comment]
Looks like We arent the only ones and its being actively worked on

https://github.com/kagemomiji/airsonic-advanced/issues/392

**Comment by Faeb35** [comment, marked as answer]
kagemoniji fixed it! just install the newest edge release and do a scan of your library

**Reply by noah1234j** [reply]
Indeed he did. Thank you kagemoniji!!!! Thanks for updating this. Does he have a link where I can make a donation to support him and this project?

**Reply by Faeb35** [reply]
I would love to support him too!
